.github/workflows/build-ls-bus.yml

```diff
@@ -31,8 +31,9 @@ jobs:
       uses: avattathil/asciidoctor-action@master
       with:
         program: "asciidoctor -D build --backend=html5 --attribute=revnumber=${{ env.REVNUMBER }} --attribute=revdate=$(date +%F) docs/index.adoc"
+      
     - name: copy images and examples
-      run: tar cfC - docs images examples | tar xfC - build
+      run: tar --exclude=headers/tests/* -c headers -C docs images examples | tar xfC - build
```

Comment:
This seems erroneous, the current step does already not include headers, so why is there a need to now include headers but exclude tests? If headers should be included in the built artefact, this should be handled in a separate step/call and PR, and the name of the step should match.

BTW the exclude syntax should not rely on globbing, but rather be `--exclude=headers/tests`, which will be more reliable and excludes the empty directory.